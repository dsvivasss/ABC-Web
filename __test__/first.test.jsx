import React from 'react';
import { render } from '@testing-library/react';
import Home from '../src/app/page';
import '@testing-library/jest-dom';

it('Dummy test', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
});