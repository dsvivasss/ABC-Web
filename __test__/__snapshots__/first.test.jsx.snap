// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dummy test 1`] = `
<div>
  <main>
    <div>
      <h1
        class="animate-fade-up from-black bg-clip-text text-center font-display text-4xl font-bold tracking-[-0.02em] [text-wrap:balance] md:text-7xl md:leading-[5rem]"
        style="animation-delay: 0.15s; animation-fill-mode: forwards;"
      >
        Registro
      </h1>
    </div>
    <div
      class="mx-auto mt-6 flex animate-fade-up items-center justify-center space-x-5 opacity-0 pb-6"
      style="animation-delay: 0.3s; animation-fill-mode: forwards;"
    >
      <a
        class="group flex max-w-fit items-center justify-center space-x-2 rounded-full border border-black bg-black px-5 py-2 text-sm text-white transition-colors hover:bg-white hover:text-black"
        href="/"
        rel="noopener noreferrer"
      >
        <p>
          Accede como aspirante
        </p>
      </a>
      <a
        class="flex max-w-fit items-center justify-center space-x-2 rounded-full border border-gray-300 bg-white px-5 py-2 text-sm text-gray-600 shadow-md transition-colors hover:border-gray-800"
        href="/company"
        rel="noopener noreferrer"
      >
        <p>
          <span
            class="hidden sm:inline-block"
          >
            Accede como empresa
          </span>
        </p>
      </a>
    </div>
    <div>
      <div>
        <h2>
          Conectamos profesionales de TI
        </h2>
      </div>
      <div
        class="mt-10 sm:mx-auto sm:w-full sm:max-w-sm"
      >
        <form
          class="space-y-6"
        >
          <div>
            <label
              class="block text-sm font-medium leading-6 text-gray-900"
              for="name"
            >
              Nombre
            </label>
            <div
              class="mt-2"
            >
              <input
                autocomplete="name"
                class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
                id="name"
                name="name"
                placeholder="  Mi nombre"
                required=""
                type="name"
              />
            </div>
          </div>
          <div>
            <label
              class="block text-sm font-medium leading-6 text-gray-900"
              for="email"
            >
              Correo
            </label>
            <div
              class="mt-2"
            >
              <input
                autocomplete="email"
                class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
                id="email"
                name="email"
                placeholder="name@domain.com"
                required=""
                type="email"
              />
            </div>
          </div>
          <div>
            <div
              class="flex items-center justify-between"
            >
              <label
                class="block text-sm font-medium leading-6 text-gray-900"
                for="password"
              >
                Contraseña
              </label>
            </div>
            <div
              class="mt-2"
            >
              <input
                autocomplete="current-password"
                class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
                id="password"
                name="password"
                placeholder="********"
                required=""
                type="password"
              />
            </div>
          </div>
          <label
            class="block text-sm font-medium leading-6 text-gray-900"
            data-headlessui-state=""
            id="headlessui-listbox-label-:r0:"
          >
            Idioma
          </label>
          <div
            class="relative mt-2"
          >
            <button
              aria-expanded="false"
              aria-haspopup="listbox"
              aria-labelledby="headlessui-listbox-label-:r0: headlessui-listbox-button-:r1:"
              class="relative w-full cursor-default rounded-md bg-white py-1.5 pr-10 text-left text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 focus:outline-none focus:ring-2 focus:ring-indigo-500 sm:text-sm sm:leading-6"
              data-headlessui-state=""
              id="headlessui-listbox-button-:r1:"
              type="button"
            >
              <span
                class="flex items-center"
              >
                <span
                  class="ml-3 block truncate"
                >
                  Español
                </span>
              </span>
              <span
                class="pointer-events-none absolute inset-y-0 right-0 ml-2 flex items-center pr-2 pt-2"
              >
                <svg
                  aria-hidden="true"
                  class="h-5 w-5 text-gray-900"
                  fill="currentColor"
                  viewBox="0 0 20 20"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    clip-rule="evenodd"
                    d="M10 3a.75.75 0 01.55.24l3.25 3.5a.75.75 0 11-1.1 1.02L10 4.852 7.3 7.76a.75.75 0 01-1.1-1.02l3.25-3.5A.75.75 0 0110 3zm-3.76 9.2a.75.75 0 011.06.04l2.7 2.908 2.7-2.908a.75.75 0 111.1 1.02l-3.25 3.5a.75.75 0 01-1.1 0l-3.25-3.5a.75.75 0 01.04-1.06z"
                    fill-rule="evenodd"
                  />
                </svg>
              </span>
            </button>
          </div>
          <div>
            <label
              class="block text-sm font-medium leading-6 text-gray-900"
              for="skill"
            >
              Habilidades
            </label>
            <div
              class="mt-2"
            >
              <input
                autocomplete="skill"
                class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
                id="skill"
                name="skill"
                placeholder="  Selecciona tus habilidades"
                required=""
                type="skill"
              />
            </div>
          </div>
          <div>
            <label
              class="block text-sm font-medium leading-6 text-gray-900"
              for="personality"
            >
              Personalidad
            </label>
            <div
              class="mt-2 pb-2"
            >
              <input
                autocomplete="personality"
                class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
                id="personality"
                name="personality"
                placeholder="  Selecciona tu personalidad"
                required=""
                type="personality"
              />
            </div>
          </div>
          <div
            class="pb-1"
          >
            <button
              class="flex w-full justify-center rounded-md bg-indigo-600 px-3 py-1.5 text-sm font-semibold leading-6 text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
              href="https://nextjs.org/learn?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
              type="submit"
            >
              Registrarse
            </button>
          </div>
        </form>
        <p
          class="mt-10 text-center text-sm text-gray-500 pt-4"
        >
          ¿Ya tienes una cuenta?
           
          <a
            class="font-semibold leading-6 text-indigo-600 hover:text-indigo-500"
            href="https://nextjs.org/learn?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
          >
            Iniciar sesión
          </a>
        </p>
      </div>
    </div>
  </main>
</div>
`;
